### ISSUE TITLE
How to read powertop output? It says my device usage is 100%

### ISSUE THREAD

**Issue by yurii-anikin** [fenrus75/powertop#105]
Hello. First of all I am sorry for silly name of the issue.

My issue is that powertop shows that my built-in fingerprint reader usage is 100% and that is scary

![image](https://user-images.githubusercontent.com/109966673/182844335-ac056e83-3643-441c-9845-b17b865b8883.png)
Does it mean that my built-in fingerprint reader (USB device: usb-device-06cb-00fc) takes ... 100% of power? How to read that? I am sorry I am little bit stressed cause my thinkpad is very hot during just regular browser surfing....

another screenshot from powertop:
![image](https://user-images.githubusercontent.com/109966673/182844669-3bf21e8c-7b12-408e-8271-89e47e4dcc27.png)

**Comment by fenrus75** [comment]
it means that this device does not participate in any power saving and also
(as a USB device that is active) will keep your CPU from using its lowest
power modes

**Comment by nikitosiusis** [comment]
And what can cause this behaviour?

**Comment by nikitosiusis** [comment]
Yes, I understand this, but where should I look into, to debug which one is doing this?:)

**Comment by st4ycool** [comment]
@fenrus75 thanks for answers. I googled more and found thinkpad users who are disabling fingerprint reader in bios, as well as other stuff like lan port, wan port(i dont have then physically, but they were in bios).
So, i guess its solver.

Very big thanks to powertop, without it I could not identify the problem!

@nikitosiusis into it's id?

**Comment by st4ycool** [comment]
@fenrus75 thanks for answers. I googled more and found thinkpad users who are disabling fingerprint reader in bios, as well as other stuff like lan port, wan port(i dont have then physically, but they were in bios).
So, i guess its solver.

Very big thanks to powertop, without it I could not identify the problem!

@nikitosiusis into it's id?

**Comment by yurii-anikin** [comment]
@fenrus75 thanks for answers. I googled more and found advices from thinkpad users who are disabling fingerprint reader in bios, as well as other stuff like LAN port, WAN port(i dont have then physically, but they were in bios). So did I.

But now I have other problem: 
Does it really means that my integrated webcamera dosn't participate in any power saving protocols? How can it be (

**Comment by fenrus75** [comment]
if you have the "lsof" tool, it often is useful in finding which things
keep what files open, you'll be looking for something keep usb devices or
input devices open
